.circleci/config.yml

```diff
@@ -1,5 +1,21 @@
 version: 2
 jobs:
+  build:
```

Comment:
If you get rid of
```yaml
   docker:
      - image: circleci/node:latest
```

and replace it with
```yaml
  build:
    machine: true
```

that'll fix the containers starting up.